src/coreclr/jit/eeinterface.cpp

```diff
@@ -569,6 +569,42 @@ const char* Compiler::eeGetClassName(CORINFO_CLASS_HANDLE clsHnd, char* buffer,
     return printer.GetBuffer();
 }
 
+//------------------------------------------------------------------------
+// eeIsClassExact: check if a class handle can only describe values
+//    of exactly one class.
+//
+// Arguments:
+//    classHnd - handle for class in question
+//
+// Return value:
+//    true if class is final and not subject to special casting from
+//    variance or similar.
+//
+// Note:
+//    We are conservative on arrays of primitive types here.
+//
+bool Compiler::eeIsClassExact(CORINFO_CLASS_HANDLE clsHnd)
```

Comment:
I'll remove this method because Jan already moved it to VM side in https://github.com/dotnet/runtime/pull/97424